src/go/build/deps_test.go

```diff
@@ -382,6 +382,7 @@ var pkgDeps = map[string][]string{
 		"golang.org/x/net/lex/httplex",
 		"internal/nettrace",
 		"net/http/httptrace",
+		"regexp",
```

Comment:
> The http2 code is bundled from golang.org/x/net/http2 so it should be modified there.

I don't see any real reason to do it that way, since like I said, its not going upstream.